cs/src/grpc/grpc.csproj

```diff
@@ -1,38 +1,26 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath32)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath32)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$(MSBuildThisFileDirectory)\..\..\build\internal\Common.Internal.props" />
+
   <PropertyGroup>
-    <ProjectGuid>{AF03BAE6-2470-4E1B-A683-3EBDCDC595FA}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFrameworks>net45;netstandard1.6</TargetFrameworks>
+    <Configurations>Debug;Release</Configurations>
     <RootNamespace>Bond.Grpc</RootNamespace>
     <AssemblyName>Bond.Grpc</AssemblyName>
     <BondRedistributable>true</BondRedistributable>
   </PropertyGroup>
+
+  <Import Project="..\..\packages\Grpc.Core.1.10.0\build\net45\Grpc.Core.targets" Condition="Exists('..\..\packages\Grpc.Core.1.10.0\build\net45\Grpc.Core.targets')" />
```

Comment:
I thought that `<PackageReference>` did this automatically.

Reply:
It does, this line should be removed.